canonical/ubuntu.com
# WD-13781 - refresh pro section on /download/amd

## Done

Refreshed Ubuntu Pro section on /download/amd

## QA

- [demo link](https://ubuntu-com-14192.demos.haus/download/amd)
- [copy doc](https://docs.google.com/document/d/1cxQnEErs8FZ3dZ_IhlfcaJ03gE_vrMJfRC-5me9IDd4/edit)
- [figma](https://www.figma.com/design/2wdbNpQgwUOVveZQYdxY7S/23.10-U.com---Download-AMD-Xilinx?node-id=180-2533&t=uvW1GSytm2g0JM7K-0)
- View the site on demo
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check that the page is correct according to figma and copy doc

## Issue / Card
[WD-13781](https://warthogs.atlassian.net/browse/WD-13781)

[WD-13781]: https://warthogs.atlassian.net/browse/WD-13781?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
